File: app/src/main/java/com/vereshchagin/nikolay/stankinschedule/about/AboutFragment.java
package com.vereshchagin.nikolay.stankinschedule.about;


import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.fragment.app.Fragment;

import com.vereshchagin.nikolay.stankinschedule.BuildConfig;
import com.vereshchagin.nikolay.stankinschedule.R;

/**
 * Фрагмент вкладки о приложении.
 */
public class AboutFragment extends Fragment {

    public AboutFragment() {
        super();
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_about, container, false);

        // установка версии приложения
        TextView version = view.findViewById(R.id.app_version);
        version.setText(String.format("Version: %s", BuildConfig.VERSION_NAME));

        return view;
    }
}

File: app/src/main/java/com/vereshchagin/nikolay/stankinschedule/ui/about/AboutFragment.kt
package com.vereshchagin.nikolay.stankinschedule.ui.about

import android.os.Bundle
import com.vereshchagin.nikolay.stankinschedule.BuildConfig
import com.vereshchagin.nikolay.stankinschedule.R
import com.vereshchagin.nikolay.stankinschedule.databinding.FragmentAboutBinding
import com.vereshchagin.nikolay.stankinschedule.ui.BaseFragment
import com.vereshchagin.nikolay.stankinschedule.utils.CommonUtils
import com.vereshchagin.nikolay.stankinschedule.utils.Constants

/**
 * Фрагмент вкладки о приложении.
 */
class AboutFragment : BaseFragment<FragmentAboutBinding>(FragmentAboutBinding::inflate) {

    override fun onPostCreateView(savedInstanceState: Bundle?) {
        binding.appVersion.text = getString(R.string.about_version, BuildConfig.VERSION_NAME)

        binding.aboutPrivacyPolicy.setOnClickListener {
            CommonUtils.openBrowser(requireContext(), Constants.PRIVACY_POLICY_URL)
        }

        binding.aboutTerms.setOnClickListener {
            CommonUtils.openBrowser(requireContext(), Constants.TERMS_CONDITIONS_URL)
        }
    }
}